### PR TITLE
Prevent wasteful thumbnail fetch requests

### PR DESCRIPTION
**Pull request by vishnukvmd**

### CODE REVIEW

File: lib/ui/huge_listview/lazy_loading_gallery.dart [hunk 1]
```diff
@@ -352,7 +352,7 @@ class _LazyLoadingGridViewState extends State<LazyLoadingGridView> {
 
   Widget _getRecyclableView() {
     return VisibilityDetector(
-      key: UniqueKey(),
+      key: Key(widget.filesInDay.first.tag),
```
Comment (ua741) [inline lib/ui/huge_listview/lazy_loading_gallery.dart:355]: Can you try archive or delete all files in a day and confirm that it doesn't break anything?

Comment (vishnukvmd) [reply lib/ui/huge_listview/lazy_loading_gallery.dart:355]: Archive worked as expected.

File: lib/ui/collections/remote_collections_grid_view_widget.dart [hunk 2]
```diff
@@ -14,6 +14,7 @@ class RemoteCollectionsGridViewWidget extends StatelessWidget {
   static const maxThumbnailWidth = 224.0;
   static const fixedGapBetweenAlbum = 8.0;
   static const minGapForHorizontalPadding = 8.0;
+  static const collectionItemsToPreload = 100;
```
Comment (ua741) [inline lib/ui/collections/remote_collections_grid_view_widget.dart:17]: why keep such high value? 10 or 15 should be more than enough, no?

Comment (vishnukvmd) [reply lib/ui/collections/remote_collections_grid_view_widget.dart]: Reduced it to 20.

**Review by ua741 — APPROVED** [review]

**Review by ashilkn — APPROVED** [review]

**Pull request merged**